assets/src/edit-story/components/popup/index.js

```diff
@@ -67,9 +67,8 @@ const Container = styled.div.attrs(
 )`
   max-height: ${({ maxHeight }) =>
     maxHeight > 0 ? maxHeight + 'px' : 'initial'};
-  overflow-y: auto;
   position: fixed;
-  z-index: 2147483646;
+  z-index: 2;
```

Comment:
Shouldn't popup be absolutely definitely above all else? Maybe with an exception of snackbar?

Reply:
Both popup and modals are using the react portals, hence the issue...